state/peerAccountsDB.go

```diff
@@ -54,7 +56,25 @@ func NewPeerAccountsDB(
 			},
 			storagePruningManager: storagePruningManager,
 		},
-	}, nil
+	}
+
+	val, err := trieStorageManager.GetFromCurrentEpoch([]byte(common.ActiveDBKey))
```

Comment:
since this is duplicate with the AccountsDB implementation, we could have had a commonDB implementation. 
Can be done in another PR (or another commit, if required)

Reply:
Fixed without the commonDB implementation.